global.json

```diff
@@ -1,5 +1,6 @@
 {
   "sdk": {
-    "version": "6.0.400"
+    "version": "8.0.0",
```

Comment:
Please use .NET SDK versions here (8.0.100) instead of .NET runtime versions (8.0.0).